src/test/projectsCreate.spec.ts

```diff
@@ -0,0 +1,204 @@
+import { expect } from "chai";
+import * as sinon from "sinon";
+
+import * as api from "../api";
+import {
+  addFirebaseToCloudProject,
+  createCloudProject,
+  ParentResource,
+  ParentResourceType,
+} from "../projectsCreate";
+import * as pollUtils from "../operation-poller";
+import { mockAuth } from "./helpers";
+
+const PROJECT_ID = "the-best-firebase-project";
+const PROJECT_NUMBER = "1234567890";
+const PROJECT_NAME = "The Best Project";
+const PARENT_RESOURCE: ParentResource = { id: "1111111111111", type: ParentResourceType.FOLDER };
+const OPERATION_RESOURCE_NAME_1 = "operations/cp.11111111111111111";
+const OPERATION_RESOURCE_NAME_2 = "operations/cp.22222222222222222";
+
+describe("projectsCreate", () => {
+  let sandbox: sinon.SinonSandbox;
+  let apiRequestStub: sinon.SinonStub;
+  let pollOperationStub: sinon.SinonStub;
+
+  beforeEach(() => {
+    sandbox = sinon.createSandbox();
+    mockAuth(sandbox);
+    apiRequestStub = sandbox.stub(api, "request");
+    pollOperationStub = sandbox.stub(pollUtils, "pollOperation");
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  describe("createCloudProject", () => {
+    it("should resolve with cloud project data if it succeeds", async () => {
+      const expectedProjectInfo = {
+        projectNumber: PROJECT_NUMBER,
+        projectId: PROJECT_ID,
+        name: PROJECT_NAME,
+      };
+      const createCloudProjectStub = createCloudProjectApiStub().resolves({
```

Comment:
Similar to the explanation from the other PR, can these stubs be defined more inline so the tests are easier to read, understand, and error in expected ways? Thanks.

Reply:
I just pushed the commit updating the tests.